rlinf/algorithms/losses.py

```diff
@@ -246,11 +269,50 @@ def compute_math_ppo_actor_loss(**kwargs):
         "approx_kl": approx_kl.detach(),
         "clip_fraction": clip_fraction.detach(),
     }
-    return policy_loss, metrics_data
+
+    # Add value loss if provided and critic is enabled
+    if (use_critic and use_value_loss and 
+        values is not None and prev_values is not None and returns is not None):
+        value_pred_clipped = prev_values + (values - prev_values).clamp(
+            -value_clip, value_clip
+        ) if value_clip is not None else values
+        error_clipped = returns - value_pred_clipped
+        error_original = returns - values
+        value_loss_clipped = huber_loss(error_clipped, huber_delta)
+        value_loss_original = huber_loss(error_original, huber_delta)
+        value_loss = torch.max(value_loss_original, value_loss_clipped)
+        
+        if value_clip is not None:
+            value_clip_indicator = (value_pred_clipped - prev_values).abs() > value_clip
+            value_clip_ratio = value_clip_indicator.float().mean()
+        else:
+            value_clip_ratio = torch.tensor(0.0, device=value_loss.device)
+        
+        value_loss = loss_agg_func(value_loss, loss_mask)
+        total_loss = total_loss + value_loss
+        
+        metrics_data.update({
+            "value_loss": masked_mean(value_loss.detach(), loss_mask),
+            "value_clip_ratio": value_clip_ratio.detach().item(),
+        })
+
+    # Add entropy loss if provided
+    if entropy is not None and entropy_bonus > 0:
+        entropy_loss = loss_agg_func(entropy, loss_mask)
+        total_loss = total_loss - entropy_bonus * entropy_loss
+        
+        metrics_data.update({
+            "entropy_loss": masked_mean(entropy_loss.detach(), loss_mask),
+        })
+
+    # Update total loss in metrics
+    metrics_data["total_loss"] = masked_mean(total_loss.detach(), loss_mask)
+    
+    return total_loss, metrics_data
```

Comment:
@guozhen1997 please help review the algorithm part, i.e., advantages.py and losses.py